packages/mocks/package.json

```diff
@@ -21,6 +21,7 @@
     "lint:fix": "eslint . --fix"
   },
   "dependencies": {
+    "@kubev2v/types": "*",
```

Comment:
it's a dev dependency, types are not needed at run time, unless we import a method ?